site/assets/js/partials/snippets.js

```diff
@@ -13,9 +13,7 @@
 
 /* global bootstrap: false */
 
-(() => {
-  'use strict'
-
+export default () => {
```

Comment:
This modification leads to a regression on Stacklitz env: when you go to https://deploy-preview-38993--twbs-bootstrap.netlify.app/docs/5.3/components/tooltips/#tooltips-on-links and click on the StackBlitz button, in the example, the tooltips don't work anymore. When replacing `export default () => {` by the previous code directly in this Stackblitz dynamic example, it works again.

Reply:
Same reason as above I guess

Reply:
I'll try to find time to try to find a more elegant solution. In the meantime, there's the hacky way:

```diff
diff --git a/site/assets/js/stackblitz.js b/site/assets/js/stackblitz.js
index 458b567ef..4b5f8cc96 100644
--- a/site/assets/js/stackblitz.js
+++ b/site/assets/js/stackblitz.js
@@ -52,7 +52,7 @@ ${htmlSnippet.trimStart().replace(/^/gm, '    ').replace(/^ {4}$/gm, '').trimEnd
 </html>
 `
 
-  const jsSnippetContent = jsSnippet ? jsSnippetFile : null
+  const jsSnippetContent = jsSnippet ? (jsSnippetFile.replace(/export default \(\) => {/, '(() => {\n  \'use strict\'') + ')()') : null // eslint-disable-line prefer-template
   const project = {
     files: {
       'index.html': markup,
```

Reply:
Doesn't seem to work and it indeed looks a lot hacky, but it seems you have pinpointed the root cause.

Must be because now the snippet partial is a module and it's not called on StackBlitz while previously it was an IIFE.

I pushed another patch which should fix the issue.

I think we are getting close :)